### DISCUSSION TITLE
Inquiry about the Release Date for SQLAlchemy 2.0.0.23

### DISCUSSION THREAD

**Discussion by jordiarellano1996** [sqlalchemy/sqlalchemy#10551] [Usage Questions] [answered]
Hello,

I hope this message finds well. I've been eagerly following the development of SQLAlchemy, and I'm excited about the upcoming release of version 2.0.0.23. I understand that release schedules can be somewhat fluid in the open-source world, but I was wondering if there's any estimated release date for this version.

I'd appreciate any information or updates you can provide regarding when we might expect SQLAlchemy 2.0.0.23 to be officially released. This information would be incredibly helpful for planning my project's dependencies.

Thank you for your hard work and dedication to the SQLAlchemy project. Your efforts are greatly appreciated.
Best regards,

J.

**Comment by zzzeek** [comment, marked as answer]
if you look at release dates in the [changelog](https://docs.sqlalchemy.org/en/20/changelog/changelog_20.html) or on [pypi](https://pypi.org/project/SQLAlchemy/#history) you'll see we release once a month and sometimes twice a month.   so 2.0.23 shouldn't be long, what specific bugfix is a blocker for you?

**Reply by jordiarellano1996** [reply]
I'm really excited about the new DBAPI support for aioodbc to connect to Microsoft SQL Server. 
I'm eagerly awaiting the next release, 2.0.23, to take advantage of it. Once it's available, it will significantly benefit our project. Thanks for your quick release schedule!
Thanks for your time,
J.
